### ISSUE TITLE
Generic OAUTH typos

### ISSUE THREAD

**Issue by atward** [sous-chefs/grafana#245]
`grafana_config_auth` parameters are prefixed with `generic_oath_` (`oath` not `oauth`)
https://github.com/sous-chefs/grafana/blob/e4bfdcb638c3204fa44df23e49a14cfd1c42e876/resources/config_auth.rb#L72-L88

This typo continues throughout resource
https://github.com/sous-chefs/grafana/blob/e4bfdcb638c3204fa44df23e49a14cfd1c42e876/resources/config_auth.rb#L134-L168

However template expected `generic_oauth_`
https://github.com/sous-chefs/grafana/blob/5b4d3b10ad0453e17655f480bc09164129dfc7af/templates/grafana.ini.erb#L582-L633

Documentation honours typo
https://github.com/sous-chefs/grafana/blob/e4bfdcb638c3204fa44df23e49a14cfd1c42e876/documentation/grafana_config_auth.md

### Expected Result:
Properties to be prefixed with `generic_oauth_`

### Actual Result:
One of
* User error, eg:  `undefined method 'generic_oath_user_info_url'`
* incorrect config
   ```
  File /etc/grafana/grafana.ini
     ×  content should include "[auth.generic_oauth]"
   ```

**Comment by xorima** [comment]
Hi @atward 

Thanks for reporting this, clearly I cannot spell :D 

Anyway pr opened to resolve this in #246, once merged will  get released, hopefully some point today

Thanks!

**Comment by damacus** [comment]
Closed via #246
